web/src/components/Layout.tsx

```diff
@@ -69,11 +69,6 @@ const Body = styled.div<{ fullWidth: boolean; disableScrollingSafari: boolean }>
         padding-left: calc((100vw - ${dimensions.maxWidth}px) / 2);
       }
     `};
-
-  @media screen and ${dimensions.smallViewport} {
-    display: flex;
-    flex-direction: column-reverse;
```

Comment:
I have no idea why we used that `column-reverse` since it means that content starts being displayed from the end of the space (and with the first elements at that end of the space but that is fine here since we only have one element inside).

Reply:
Maybe it got broken when I removed the feedback modal from here... wouldn't know why though.